src/Persistence.php

```diff
@@ -137,27 +137,27 @@ public function atomic(\Closure $fx)
      *     'is_married'=>1
      *   ]
      */
-    public function typecastSaveRow(Model $m, array $row): array
+    public function typecastSaveRow(Model $model, array $row): array
     {
         $result = [];
-        foreach ($row as $key => $value) {
+        foreach ($row as $fieldName => $value) {
             // We have no knowledge of the field, it wasn't defined, so
             // we will leave it as-is.
-            if (!$m->hasField($key)) {
-                $result[$key] = $value;
+            if (!$model->hasField($fieldName)) {
+                $result[$fieldName] = $value;
 
                 continue;
             }
 
             // Look up field object
-            $f = $m->getField($key);
+            $field = $model->getField($fieldName);
```

Comment:
yes, `field` is object, `fieldName` is string, would be great to fix in Model etc.